doc/foreign-code.rst

```diff
@@ -0,0 +1,64 @@
+******************************
+Dealing with foreign libraries
+******************************
+
+The OCaml programming language allows to interface libraries written
+in foreign languages such as C. This section explains how to do this
+with Dune. Note that it does not cover how to write the C stubs
+themselves, this is covered by the OCaml manual: LINK TO MANUAL.
```

Comment:
That would be https://caml.inria.fr/pub/docs/manual-ocaml/intfc.html

Reply:
Fixed. I wrote this documentation in the plane and their proxy was blocking caml.inria.fr, can you believe it...